liquidctl/driver/nvidia.py

```diff
@@ -0,0 +1,171 @@
+"""liquidctl drivers for NVIDIA graphics cards.
```

Comment:
You sort of mentioned this in one of your messages but why is this driver called `nvidia`? Its intention is not to store the drivers for all NVIDIA graphics cards in this file is it?

Reply:
The basic issue is that I don't think it's sensible to have a bunch of single-driver modules, especially when there are significant similarities between the drivers.  _(Module-hell is also not idiomatic in Python)._

Eventually the we'll need to have one module for each "generation", so each module has one to ten drivers (each board partner has a few variants of each GPU, sometimes with different protocols).  But since we're going to be much more strict with adding these drivers, because of the increased associated risks, I'm betting it's going to take a long time until we have more than ten NVIDIA drivers in total.

For now I think we should keep all of them in a single `nvidia` module.  When that starts to get too big we'll split it into `pascal`, `turing`, `ampere`, etc., and at that point `nvidia` can simply re-export all drivers in a single namespace, which is more convenient for API users and also retains backwards compatibility.